HW7.py

```diff
@@ -0,0 +1,44 @@
+phone_book = {
+    'Igor': 132434,
+    'Diana': 254345,
+}
+
+print (phone_book.keys())
+
+while True:
+    user_input = input('Enter a command: ')
+    split_input = user_input.split()
+   
+    command = split_input[0]
+    name = split_input[1]
```

Comment:
The command `list` raises an error here, because there are not enough value for this index. You can add some check here, like this:
```
if len(split_input) > 1:
    name = split_input[1]
else:
    name = None

Reply:
done